css/small.css

```diff
@@ -0,0 +1,30 @@
+#header {
+  height: 50px;
+}
+
+#logo {
+  height: 20px;
+}
+
+#head-container {
+  margin-bottom: 40px;
+  width: 100%;
+  position: fixed;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.